src/pages/ArticleList.jsx

```diff
@@ -0,0 +1,18 @@
+import { ArticleCard } from '../components/ArticleCard';
+
+export const ArticleList = ({ articleList }) => {
+  return (
+    <div className="article-list">
```

Comment:
does this need to be a div? Is there anything that is a bit more semantic that you could use?